Dockerfile

```diff
@@ -28,33 +36,64 @@ LABEL summary="$SUMMARY" \
     com.redhat.component="python38-docker" \
     name="Koku" \
     version="1" \
-    maintainer="Red Hat Cost Management Services"
+    maintainer="Red Hat Cost Management Services <cost-mgmt@redhat.com>"
 
-USER root
+# Very minimal set of packages
+# glibc-langpack-en is needed to set locale to en_US and disable warning about it
+# gcc to compile some python packages (e.g. ciso8601)
+# shadow-utils to make useradd available
+RUN INSTALL_PKGS="python38 python38-devel glibc-langpack-en gcc shadow-utils" && \
+    microdnf --nodocs -y upgrade && \
+    microdnf -y --setopt=tsflags=nodocs --setopt=install_weak_deps=0 install $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    microdnf -y clean all --enablerepo='*'
 
-# Get latest packages
-# RUN dnf -y upgrade
+WORKDIR ${HOME}
```

Comment:
Are we breaking with convention here or is `/opt/app-root` no longer relevant?

Reply:
Honestly, I'm not sure what convention is. `/opt/app-root` seemed more related to the old s2i images, but I could be wrong.

Reply:
I'll move things into `/opt/koku`.